tests/data/cpu_template_helper/fingerprint_AMD_MILAN_4.14host.json

```diff
@@ -1,9 +1,9 @@
 {
-  "firecracker_version": "1.6.0-dev",
-  "kernel_version": "4.14.327-246.539.amzn2.x86_64",
+  "firecracker_version": "1.7.0-dev",
```

Comment:
I assume the version update is not part of the fix but just to keep the files updated, is that correct?
and if that is the case, how often do we need to update these files?